war/src/main/js/add-item.js

```diff
@@ -107,7 +107,6 @@ $.when(getItems()).done(function(data) {
           btn.removeClass('disabled');
           btn.prop('disabled', false);
         }
-        btn.focus();
```

Comment:
👎 The current behavior was deeply discussed with @gusreiber. At any rate, it seems there are a lot of code in progress. I'll wait until something more stable.